doc/api/next_api_changes/removals/18070-DS.rst

```diff
@@ -0,0 +1,14 @@
+rcsetup removals
+~~~~~~~~~~~~~~~~
+
+The following functions in `~.rcsetup` have been removed:
```

Comment:
```suggestion
The following functions in `matplotlib.rcsetup` have been removed:
```
For clarity, I'd give the full name here.

*Side remark:* `~` trims the display of the given reference to the last part. If only one part is given like in `~.rcsetup`, it does not have any effect and thus one can simplify to `.rcsetup`.